library/clinical_rules.php

```diff
@@ -589,8 +596,159 @@ function test_rules_clinic_batch_method($provider = '', $type = '', $dateTarget
     }
 }
 
+function rules_clinic_get_providers($billing_facility, $pat_prov_rel)
+{
+    $results = [];
+    // TODO: @adunsulag do we need to batch the provider processing here?  Its unlikely to have more than 50-100 providers...
```

Comment:
no need to batch for now; the batching for patients was done since it literally took days for when patients over 10000 because it kept getting slower with each patient. The patient batching removed this issue.